packages/dashboard/src/components/tests/appbar.test.tsx

```diff
@@ -26,32 +28,49 @@ describe('AppBar', () => {
   let toggleShowOmniPanel: jest.Mock;
   let showSettings: jest.Mock;
   let showHelp: jest.Mock;
+  let tooltips: boolean;
 
   beforeEach(() => {
     toggleShowOmniPanel = jest.fn();
     showSettings = jest.fn();
     showHelp = jest.fn();
+    tooltips = true;
   });
 
   test('renders correctly', () => {
-    const root = buildWrapper(toggleShowOmniPanel, showSettings, showHelp);
+    const root = buildWrapper(toggleShowOmniPanel, showSettings, showHelp, tooltips);
     expect(root).toMatchSnapshot();
   });
 
+  test('renders tooltips when it is enabled', () => {
+    const root = buildWrapper(toggleShowOmniPanel, showSettings, showHelp, tooltips);
+    expect(root.find('#omnipanel-tooltip').length > 0).toBeTruthy();
+    expect(root.find('#setting-tooltip').length > 0).toBeTruthy();
+    expect(root.find('#help-tooltip').length > 0).toBeTruthy();
+  });
+
+  test('does not render tooltips when disabled', () => {
+    tooltips = false;
+    const root = buildWrapper(toggleShowOmniPanel, showSettings, showHelp, tooltips);
+    expect(root.find('#omnipanel-tooltip').length <= 0).toBeTruthy();
+    expect(root.find('#setting-tooltip').length <= 0).toBeTruthy();
+    expect(root.find('#help-tooltip').length <= 0).toBeTruthy();
```

Comment:
Do you think it'll be possible to use `exists() `here?

Reply:
yep this would work too!